plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -1397,7 +1397,7 @@ def main():
             module.fail_json(msg='VM with name = %s does not exist in cluster' % name)
         vm = proxmox.get_vm(vmid)
         if not name:
-            name = vm['name']
+            name = vm.get('name', '(no name)')
```

Comment:
As a matter of wordsmithing preference, I would go with `(unnamed)` but the effect is exactly the same, I am not pushing it.